### PR TITLE
[2.4]PANDARIA: upgrade to istio 1.8.6

### PR DESCRIPTION
**Pull request by aiwantaozi**
Issue: https://github.com/cnrancher/pandaria/issues/1273

### CODE REVIEW

**Comment by guangbochen** [comment]
can u specify why those configurations are removed from this PR?

**Comment by aiwantaozi** [comment]
For the first commit, we remove some gateway configure.
We used to configure port 80 and 443 under `ports`, port 15021 and 15443 under `ingressPorts`, base on the [original chart](https://github.com/istio/istio/blob/4d6e76c457a653329aae2aabeb2a5a694484b478/manifests/charts/gateways/istio-ingress/values.yaml#L15-L31), these four ports should under ports, we already configured the correct default ports under
 [sub charts](https://github.com/cnrancher/system-charts/blob/27309afeeb6f3a00b2907ca46c32e018177e346d/charts/rancher-istio/1.8.5000/charts/gateways/charts/istio-ingressgateway/values.yaml#L19-L35), when user customizes these value, UI will pass them, so these should be removed.

File: charts/rancher-istio/1.8.5000/values.yaml [hunk 1]
```diff
@@ -39,32 +39,6 @@ gateways:
       app: istio-ingressgateway
```
Comment (GGGitBoy) [inline charts/rancher-istio/1.8.5000/values.yaml]: labels are also configured in sub charts，look like can be removed.

Comment (aiwantaozi) [reply charts/rancher-istio/1.8.5000/values.yaml]: updated

**Comment by GGGitBoy** [comment]
LGTM

**Pull request merged**